### DISCUSSION TITLE
nullable bool property in Csla 4.xx

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3356] [Questions] [answered]
I am trying to create a nullable bool property in a Csla 4.xx application and when I try to assign a default value of null in RegisterProperty. I am getting this error

![image](https://github.com/MarimerLLC/csla/assets/19640475/b84678da-3f70-467a-aae9-a8a6ac0f31b4)

**Comment by kcabral817** [comment]
Can you post the code for the property?

**Reply by Chicagoan2016** [reply]
```cs
  public static readonly PropertyInfo<bool?> IsVoidProperty = RegisterProperty<bool?>(c => c.IsVoid);
        public bool? IsVoid
        {
            get { return GetProperty(IsVoidProperty); }
            private set { LoadProperty(IsVoidProperty, value); }
        }
```

**Comment by Chicagoan2016** [comment, marked as answer]
I found out it's a .NET limitation. Nothing to do with Csla : )
